3 reports.sql

```diff
@@ -0,0 +1,49 @@
+/*1) I offer deep cleans for each customer on request. Deep cleans take twice as long as normal cleans.
+   I charge 50% more than a regular clean in that house. 
+   For this I need to see the customers phone number, last name and the amount of money they will have to pay.
+   Please do it in this format:  last name, first name (phone number) - price of service.
+*/
+SELECT DeepCleanCustomers= CONCAT(c.LastName, ' ', c.FirstName, ' (',c.PhoneNumber, ') - ', (c.PricePerHour*c.TimeinHours)  )
+FROM CleaningCompany C
+where c.PricePerHour >=35 
```

Comment:
No need for where clause as the data in table are for regular cleans only.

Reply:
took out the where clause and add to the select to multiply by 1.5 to show the price would charge for a deepclean